.eslintignore

```diff
@@ -10,3 +10,5 @@ coverage/*
 /src/main/types/
 jest.*config.js
 *.min.js
+/src/main/assets/js/jquery.js
```

Comment:
I believe JQuery should already be bundled as part of the package.json.

I think we should be able to remove this + the jquery file below

Reply:
I was not able to import already installed jQuery package into page.
I tried all importing suggestions from https://www.npmjs.com/package/jquery and none of them worked.

Reply:
Ah ok. Are these two files the one's from the @ministryofjustice package? I'm wondering if we can copy and bundle these as part of webpack build rather than needing to store them as files in the repo separately. (We do this currently with images in the webpack config files)